README.md

```diff
@@ -8,52 +8,65 @@ A loading indicator component for EmberJS that animates across the top of the vi
 
 ## Installation
 
-`npm install ember-cli-loading-slider --save-dev`
+`ember install ember-cli-loading-slider`
 
 ## Usage
 
 Add the component to your application template:
 
-    {{loading-slider isLoading=loading duration=250}}
-
-Alternatively you may try the more complex mode. In this mode an array of colors
-is required `hexColorsArray: ['#000', '#fff']`:
-
-    {{loading-slider isLoading=loading expanding=true color=hexColorsArray}}
+    {{loading-slider duration=250}}
```

Comment:
@mszoernyi elaborating on the two ways to use it could be handy. While the service is a nice way to change the state I feel like the template approach will get used quite often also